client-app/src/Components/DonatedItemDetails.tsx

```diff
@@ -87,6 +93,9 @@ const DonatedItemDetails: React.FC = () => {
                         <div className="section-header">
                             <AssignmentTurnedInIcon className="icon" />
                             <h2>Donated Item Status</h2>
+                            <button onClick={handleAddNewDonationClick}>
+                                Add New Status
+                            </button>
                         </div>
```

Comment:
@truffer11 Thanks for adding this button, even though it was requested after the sprint had already started